tests/ra_aid/proc/test_interactive.py

```
@@ -168,3 +169,27 @@ def test_tty_available():
         b"/dev/pts/" in output_cleaned or b"/dev/ttys" in output_cleaned
     ), f"Unexpected TTY output: {output_cleaned}"
     assert retcode == 0
+
+
+def test_interactive_input():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_interactive_input`

Add return type annotation: `None`

(ANN201)